### ISSUE TITLE
AdLoox Ad Server: Flaky test

### ISSUE THREAD

**Issue by patmmccann** [prebid/Prebid.js#9776]
Many pr's failing to pass circle ci; main issue appears to be an AdLoox ad server test

https://app.circleci.com/pipelines/github/prebid/Prebid.js/15291/workflows/a8f83393-c655-41f4-8ed2-25edfcf6bc87/jobs/27920

https://github.com/prebid/Prebid.js/blob/81fd1112be520d40f0dd36d3acb6876a0328f816/test/spec/modules/adlooxAdServerVideo_spec.js#L257 claims there is no network traffic; than why are there so many time out issues? @jimdigriz  would love your feedback

**Comment by jimdigriz** [comment]
My guess is something went wrong with the [`ajax.js` catch block](https://github.com/prebid/Prebid.js/blob/0beb9a8a8766457478813f85e58867362165aa58/src/ajax.js#L94-L97); my [original fix or the IE11 edge case](https://github.com/prebid/Prebid.js/commit/7a2ca923b79121c34447cec594060d07aae05427) may not be correct or good enough.

Does this only impact a single browser (this one shows Edge)? Maybe XHR does something crazy on Edge for expired Blob URLs like it did for IE11?

**Comment by jimdigriz** [comment]
Looks to again be Edge 11 for https://github.com/prebid/Prebid.js/pull/9728#issue-1646085704

**Comment by patmmccann** [comment]
You're right, appears to always be edge https://app.circleci.com/pipelines/github/prebid/Prebid.js/15343/workflows/0fda394f-e7ca-476f-9cd7-4011452a1a42/jobs/28001
